packages/starlight/components/Head.astro

```diff
@@ -18,7 +18,7 @@ const headDefaults: z.input<ReturnType<typeof HeadConfigSchema>> = [
 	{ tag: 'meta', attrs: { charset: 'utf-8' } },
 	{
 		tag: 'meta',
-		attrs: { name: 'viewport', content: 'width=device-width, initial-scale=1' },
+		attrs: { name: 'viewport', content: 'width=device-width' },
```

Comment:
While MDN might claim this is the default in practice it does not seem to be the case and without it I have seen sites break on mobile:

```suggestion
		attrs: { name: 'viewport', content: 'width=device-width, initial-scale=1' },
```

Reply:
I looked so many different resources and the only reason that it's needed was on Safari when you change orientation it's not changing the width but it's doing zoom-out, I don't have iPhone so I can't test it, but I'd like to know if it's still needed in 2024 (like a test on Safari) but still I'm okay with `initial-scale=1` to be in the codebase, so I can accept the suggestion. Thank you!